Database/Corrections/sodQuestFixes.lua

```diff
@@ -245,6 +245,11 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.finishedBy] = {nil,{410168}},
             [questKeys.requiredRaces] = raceIDs.UNDEAD,
         },
+        [78229] = {
+            [questKeys.preQuestGroup] = {90088,90089,90090},
+            [questKeys.zoneOrSort] = sortKeys.DRUID,
+            [questKeys.objectivesText] = {"To start this quest, bring 1 symbol of the owl to Loganaar in Moonglade. You need to bring at least 1 symbol, but you can bring all three at once."},
```

Comment:
```suggestion
            [questKeys.objectivesText] = {"Bring symbols of the three owls to Loganaar in Moonglade. NOTE: To accept this quest, you need to bring at least one symbol, but you can also bring all three at once."},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5378/commits/8485acaea2c00c71ca578d6e7e88c89694e5daef